static/js/containers/SettingsPage.js

```diff
@@ -0,0 +1,101 @@
+// @flow
+/* global SETTINGS: false */
+import React from 'react';
+import { connect } from 'react-redux';
+import Loader from 'react-loader';
+import Grid, { Cell } from 'react-mdl/lib/Grid';
+
+import Jumbotron from '../components/Jumbotron';
+import {
+  startProfileEdit,
+  FETCH_PROCESSING,
+} from '../actions/index';
+import ProfileFormContainer from './ProfileFormContainer';
+import PrivacyForm from '../components/PrivacyForm';
+import ProfileProgressControls from '../components/ProfileProgressControls';
+import {
+    combineValidators,
+    privacyValidation,
+} from '../util/validation';
+import { getPreferredName } from '../util/util';
+
+class SettingsPage extends ProfileFormContainer {
```

Comment:
same comment here, would you mind adding a `props` declaration? 

something like:

```js
import type { Profile } from '../flow/profileTypes';
import type { UIState } from '../reducers/ui';

class SettingsPage extends ProfileFormContainer {
  props: {
    profiles: {[key: string]: Profile}, 
    dispatch: Function,
    ui:       UIState,
  };

  ...
}
```